### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by jsug9**
**Checks:**
- Page titles: No issue found
- Image text alternatives: Fixed
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/style.css#L220

- Text headings: Fixed
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/index.html#L51
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/index.html#L178

- Color contrast: Fixed
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/style.css#L2
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/style.css#L68
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/style.css#L77
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/style.css#L295
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/style.css#L328
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/style.css#L347
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/style.css#L404
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/style.css#L424

- Resize: Fixed
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/style.css#L297

- Interaction: Fixed (Add labels and aria-labels)
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/index.html#L180
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/index.html#L181
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/index.html#L182
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/index.html#L238
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/index.html#L240
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/index.html#L242
https://github.com/jsug9/Portfolio/blob/c49048280a88f3e0e5f755b2b2a99b850df3a711/style.css#L477

- Moving content:  No issue found
- Multimedia:  No issue found
- The basic structure of the page: No issue found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -48,11 +48,11 @@ <h1>
...
         <div class="connect-icons">
-          <h4 class="connect-title">LET'S CONNECT</h4>
+          <p class="connect-title">LET'S CONNECT</p>
           <ul class="icons-list">
-            <li><a href="https://www.linkedin.com/in/augustogalindo/" target="_blank" class="fa fa-linkedin" rel="noopener noreferrer"></a></li>
-            <li><a href="https://github.com/jsug9" target="_blank" class="fa fa-github" rel="noopener noreferrer"></a></li>
-            <li><a href="https://twitter.com/AugustoAl2" target="_blank" class="fa fa-twitter" rel="noopener noreferrer"></a></li>
+            <li><a href="https://www.linkedin.com/in/augustogalindo/" target="_blank" class="fa fa-linkedin" rel="noopener noreferrer" aria-label="Linkedin"></a></li>
+            <li><a href="https://github.com/jsug9" target="_blank" class="fa fa-github" rel="noopener noreferrer" aria-label="Github"></a></li>
+            <li><a href="https://twitter.com/AugustoAl2" target="_blank" class="fa fa-twitter" rel="noopener noreferrer" aria-label="Twitter"></a></li>
```
Comment (gemmen29) [inline index.html:55]: Please add a report on each point that exists in the project requirement in the pull request description for example:
- page title: fixed issue or no issue found.
kindly list all points you have checked and the result of the check-in in the PR description.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @jsug9 and @kemigabocatherine, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
**Hi @team**, 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] As requested by the previous reviewer, you have to add the report that exists in the project in the pull request description, If present then uses the link to denote that part otherwise write no issue found. For ex:-
```
Page Title: [Link]
Image text alternatives: [Link]
Text Heading: no issues found
Color Contrast :[Link]
Resize - no issues found
Interaction - no issues found
Moving content - no issues found
Multimedia - no issues found
The basic structure of the page - no issues found
```
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Hamzaoutdoors — APPROVED** [review]
Hi team

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**